anaconda.spec.in

```diff
@@ -131,6 +131,9 @@ Requires: systemd
 Requires: python3-pid
 Requires: python3-ordered-set >= 2.0.0
 
+# Required by the systemd service anaconda-fips.
+Requires: crypto-policies
```

Comment:
`/usr/bin/update-crypto-policies` is in `crypto-policies-scripts` in Fedora.

Reply:
Fixed.

Reply:
I have changed it back, because this is a branch for RHEL 8.